src/frontend/components/modal/modals/curval/index.js

```diff
@@ -1,10 +1,14 @@
-import { getComponentElements, initializeComponent } from 'component'
+//import { getComponentElements, initializeComponent } from 'component'
+import { initializeComponent } from 'component';
+import CurvalModalComponent from './lib/component';
 
-export default (scope) => {
-    if (!getComponentElements(scope, '.modal--curval').length) return;
+export default (scope) => initializeComponent(scope, '.modal--curval', CurvalModalComponent);
 
-    import(/* webpackChunkName="modal" */ "./lib/component")
-        .then(({ default: CurvalModalComponent }) => {
-            initializeComponent(scope, '.modal--curval', CurvalModalComponent)
-        });
-}
+//export default (scope) => {
```

Comment:
If this is no longer needed (?) can it be removed completely and/or comments added?